bluecellulab/cell/core.py

```diff
@@ -51,10 +51,12 @@
 class Cell(InjectableMixin, PlottableMixin):
     """Represents a Cell object."""
 
+    max_id = 0
```

Comment:
The name is misleading, no? This is not the maximum id. Maybe next_id, or something else?

Reply:
Oh good point, how about the last_id (for the last given id)?

Reply:
sounds good